products/zoningtaxlots/models/intermediate/_int_models.yml

```diff
@@ -0,0 +1,79 @@
+version: 2
+
+models:
+  - name: int__commercialoverlay
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - dtm_id
+            - overlay
+            - segbblgeom
+            - row_number
+  - name: int__inwoodrezooning
+    columns:
+      - name: bbl
+        data_type: string
+        tests:
+          - not_null
+  - name: int__inzonechange
+    columns:
+      - name: dtm_id
+        data_type: string
+        tests:
+          - not_null
+          - unique
+  - name: int__limitedheight
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - bbl
+            - dtm_id
+            - lhlbl
+            - segbblgeom
+            - row_number
+  - name: int__specialpurpose
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - bbl
+            - dtm_id
+            - sdlbl
+            - segbblgeom
+            - row_number
+  - name: int__validdtm
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - bbl
+            - dtm_id
+  - name: int__zoningdistricts
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
```

Comment:
I had originally put this in #928, but I think actually at this point already restrict these tests a bit. The more columns are added to this, the less stringent it is. So for each of these, think about conceptually and maybe query - what actually __should__ identify a unique row here?

Reply:
Just to rephrase - this test is essentially declaring a compound unique key/identifier. So thinking about what columns rows are defined/identified by vs what columns are more properties of the entity defined by the row

Reply:
@fvankrieken I agree! For those where we use `row_number` to pivot, I think it probably makes sense to just test the unique combinations of `dtm_id`, `bbl`, `row_number`, and the pivoting condition column if there are any (for example some specified `perbblgeom >= 10`)?

Reply:
So because of dtm weirdness, bbl isn't actually unique. So let's use `dtm_id` and __not__ `bbl`. 

`row_number` feels more of an attribute - really, these rows are mappings of dtm_id to whatever entity is represented in the other table. So in this case, I would say just `dtm_id` and `zonedist`. I don't know that this will be valid for all other tables (with their analagous columns like "zoning_map" for int__zoningmapindex, but we should do this pair of columns when possible, and then when needed (as in, if this combo is NOT unique), then include row_number as well